rll-examples/security-updates.sh

```diff
@@ -6,18 +6,53 @@
 # ...
 #
 
+# Run sudo apt-get|yum with retries if errors occur.
+# $@: full APT-GET(8) or yum(8) command
+#
+function package_handler() {
```

Comment:
Because this is only running `sudo [apt-get|yum] args...` it should be listed with something about packages.  Perhaps `package_wrapper`?  Discussed earlier, do we want to have this function with a retry for any command line call?